R/tostring.R

```diff
@@ -130,9 +130,22 @@ setMethod("toString", "MatrixPrintForm", function(x,
   cell_widths_mat <- .calc_cell_widths(mat, widths, col_gap)
   ## wrap_string calls strwrap, which destroys whitespace so we need to make
   ## sure to put the indents back in
+
+  # See if indentation is properly set
+  ind_from_mf <- mf_rinfo(mat)$indent > 0
+  nlh <- mf_nlheader(mat)
   old_indent <- gsub("^([[:space:]]*).*", "\\1", mat$strings[, 1])
```

Comment:
We can't rely on this as pointed out by the related bug you filed. We should be using `ind_from_mf` throughout instead of `old_indent` (the way it was previously calculated)

Reply:
Yes, this is causing the latest error I filed. I am working on it, bc indent does not keep track of how many spaces I think and I am searching what does